writers/influxdb/messages.go

```diff
@@ -0,0 +1,11 @@
+package influxdb
+
+import "github.com/mainflux/mainflux/writers"
+
+// Writer represents writer implementation to the InfluxDB.
+type Writer interface {
```

Comment:
See the remark given above regarding the instantiation of the underlying influxdb client.